sound/soc/sof/mediatek/mt8186/mt8186.c

```diff
@@ -628,7 +628,113 @@ static const struct sof_dev_desc sof_of_mt8186_desc = {
 	.ops = &sof_mt8186_ops,
 };
 
+static struct snd_soc_dai_driver mt8188_dai[] = {
+{
+	.name = "SOF_DL2",
+	.playback = {
+		.channels_min = 1,
+		.channels_max = 2,
+	},
+},
+{
+	.name = "SOF_DL3",
+	.playback = {
+		.channels_min = 1,
+		.channels_max = 2,
+	},
+},
+{
+	.name = "SOF_UL4",
+	.capture = {
+		.channels_min = 1,
+		.channels_max = 2,
+	},
+},
+{
+	.name = "SOF_UL5",
+	.capture = {
+		.channels_min = 1,
+		.channels_max = 2,
+	},
```

Comment:
It wouldn't hurt to add a comment on the naming conventions, for 8816 you have SOF_DL1/2 and SOF_UL1/2, not it's DL2, DL3, UL4, UL5, it's not quite clear why different names are chosen.

Reply:
I don't know what a virtual BE means, so yes you probably want to elaborate on what these concepts are.

Reply:
For the detail, please refer to the diagram.
https://github.com/thesofproject/linux/pull/3122#discussion_r704180504
SOF_DLx and SOF_ULx are created for the pipeline define in topology.

I'll add some comments before the snippet.